Updating the SIG Apps goals to reflect current state

mattfarina opened the pull request:
/cc @prydonius @michelleN 

While prepping for the SIG Apps update at Kubecon I noticed the goals were a bit dated and could use a little grammar love.

michelleN commented:
/approve

prydonius commented on sig-apps/README.md:
```diff
@@ -27,24 +27,27 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=hn23Z-vL_
 
 * The Helm Developer call is [Thursday at 16:15 UTC](https://zoom.us/j/4526666954). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1elWRfvH3AkHdr8pOaqyPbqSZ6ONR-l1Sb9_gapqh8ZA/edit).
-* Charts Chat is [Thursday at 18:00 UTC](https://zoom.us/j/166909412). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
+* Charts Chat is every other [Thursday at 18:00 UTC](https://zoom.us/j/166909412). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
```
The day and time is likely to move, but we can update this at a later stage. FWIW, right now it looks like we'll move to Tuesdays at 9AM PT. I thought we were also moving away from specifying times in UTC?

mattfarina commented:
@prydonius When we move the charts meeting we can update it.

I updated the time on the helm and charts meeting to be based on PT and fixed the timezone conversion links.

prydonius commented:
thanks
/lgtm
